### PR TITLE
SS-585  network fixes and adding seccomp profile

### PR DESCRIPTION
**Pull request by hamzaimran08**
## Description

The PR refers to the following Jira issues: [SS-585](https://scilifelab.atlassian.net/browse/SS-585), [SS-559](https://scilifelab.atlassian.net/browse/SS-559), [SS-309](https://scilifelab.atlassian.net/browse/SS-309) and [SS-619](https://scilifelab.atlassian.net/browse/SS-619).

The shiny proxy app should now be running fine with the network policy enabled as well. Have added network policies for shinyproxy in the serve-charts repo. The pods also now have securityContexts assigned to the, and run as non root. Seccomp annotations/contexts have been added to all the charts. The profile is the Runtime default that is already setup in kubernetes. It blocks around 64 system calls.

## Types of changes

To indicate the type of change (such as bugfix or new feature), use a github pull request label.

## Checklist

_If you're unsure about any of the items below, don't hesitate to ask. We're here to help!
This is simply a reminder of what we are going to look for before merging your code._

- [x] This pull request is against **develop** branch (not applicable for hotfixes)
- [x] I have included a link to the issue on GitHub or JIRA (if any)
- [ ] I have included migration files (if there are changes to the model classes)
- [ ] I have included, reviewed and executed tests (unit and end2end) to complement my changes
- [ ] I have updated the related documentation (if necessary)
- [ ] I have updated the release notes (releasenotes.md)
- [x] I have added a reviewer for this pull request
- [x] I have added myself as an author for this pull request
- [ ] In the case I have modified settings.py, then I have also updated the studio-settings-configmap.yaml file in serve-charts

## Further comments

Anything else you think we should know before merging your code!

### CODE REVIEW

**Review by sandstromviktor — APPROVED** [review]
Looks good to me! I've tried to deploy a few apps and it works fine!

File: fixtures/projects_templates.json [hunk 1]
```diff
@@ -61,6 +64,9 @@
       "slug": "default",
       "template": {
         "apps": {
+          "Project Network Policy": {
```
Comment (sandstromviktor) [inline fixtures/projects_templates.json:67]: What is the naming convention here? The other "apps" are called `project-vol` and `project-minio`. Perhaps this should be `project-netpolicy` or something?

**Review by sandstromviktor — APPROVED** [review]
Great job. Everything seems to work, except that you must delete the network policy if you deploy Serve using Docker. This is expected behavior.

File: fixtures/projects_templates.json (hunk 1 above)
Comment (hamzaimran08) [reply fixtures/projects_templates.json]: Goo point. I have updated this!

Comment (sandstromviktor) [reply fixtures/projects_templates.json]: Ok nice! Looks good

**Pull request merged**